it/utils_for_tests.go

```diff
@@ -255,6 +256,12 @@ func SetupApp(t *testing.T, bundle *smith_v1.Bundle, serviceCatalog, createBundl
 		cli := smithClient.SmithV1().Bundles(cfg.Namespace)
 		b, err := cli.Get(bundle.Name, meta_v1.GetOptions{})
 		if err == nil {
+			if t.Failed() {
```

Comment:
If an integration test fails, it's useful to see the current state of the bundle.